core/src/test/java/com/appintro/core/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.appintro.core
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
This can be removed, it's causing tests to fail since jUnit is not a dependency anymore